magento/pwa-studio
# Merge directory state and actions into checkout

## Description

Merges directory state, actions, and reducers into checkout. This gets us closer to being able to pull off `countries` data from the checkout slice.

## Related Issue





Closes #1684.

## Verification Steps

1. Open checkout, you should notice a `GET_COUNTRIES` action in console.
2. Complete checkout, you should get no errors.
3. Open another checkout. You should _not_ see a `GET_COUNTRIES` action (because we preserve the `countries` state.)

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
